telco-examples/edge-clusters/README.md

```diff
@@ -27,7 +27,16 @@ You need to modify the following values in the `telco-edge-cluster.yaml` file:
 
 You need to modify the following folder:
 
-- `base-images` - To include inside the `SL-Micro.x86_64-6.0-Base-RT-GM2.raw` image downloaded from the SUSE Customer Center.
+- `base-images` - To include inside the raw image generated with the following Kiwi process:
+
+```
+mkdir output
+sudo podman run --privileged -v /etc/zypp/repos.d:/micro-sdk/repos/ -v $(pwd)/output:/tmp/output -it registry.opensuse.org/isv/suse/edge/factory/images_16.0/kiwi-builder:10.2.12.0 build-image -p Base-RT-SelfInstall
+```
+
+In the output folder, you will find the raw image to be copied to the base-image folder and used as a reference in the `eib/telco-edge-cluster.yaml` file.
```

Comment:
```suggestion
The resulting raw image needs to be copied over to the `base-image` folder and used as a reference in the `eib/telco-edge-cluster.yaml` file:

```
cp $(pwd)/output/*.raw base-images/
```
```


I'm not sure why GitHub doesn't like the format

Reply:
ok makes sense to reduce the content. Let me change it